releasing/centos.dockerfile

```diff
@@ -0,0 +1,52 @@
+ARG _VERSION_
```

Comment:
BTW -- Ideally these commands would be in the README and pull into these files in some way (maybe using tuttest?).

Reply:
Now, the list of supported base images is located in `releasing/supported_bases.txt`. Rather than having that content in the README (which requires parsing Markdown; and that's not fancy), I would just add a link from the README to that file.